src/tools/BaseMoveTool.ts

```diff
@@ -0,0 +1,58 @@
+import { GlobalStore } from "@/store";
+import BaseTool from "./BaseTool";
+
+/**
+ * BassMoveTool handles the basics of panning the field when meta/alt key is pressed.  Derived tools implement onMouseDown/Up/Move to do tool specific actions.
+ */
+export default abstract class BaseMoveTool {
+  /**
+   * Approximates coordinate on the two step grid
+   *
+   * @param coordinate - Either the x or y coordinate
+   */
+  static enablePan(enable: boolean): void {
+    const grapherSvgPanZoom: SvgPanZoom.Instance | undefined =
+      GlobalStore.state.grapherSvgPanZoom;
+    if (grapherSvgPanZoom === undefined) {
+      throw new Error("There is no grapher pan zoom instance");
+    }
+    if (enable) {
+      grapherSvgPanZoom.enablePan();
+    } else {
+      grapherSvgPanZoom.disablePan();
+    }
+  }
+
+  onMouseDown(event: MouseEvent): void {
+    BaseTool.updateInvertedCTMMatrix();
+
+    if (event.metaKey) {
```

Comment:
Can you update this to also include the control key? The meta key on windows and linux sometimes removes focus from the browser, so the control key is needed

Reply:
will do.